packages/desktop-client/src/components/transactions/TransactionsTable.jsx

```diff
@@ -562,17 +564,40 @@ function PayeeCell({
         }
       }}
       formatter={() => getPayeePretty(transaction, payee, transferAcct)}
-      unexposedContent={props => (
-        <>
-          <PayeeIcons
-            transaction={transaction}
-            transferAccount={transferAcct}
-            onNavigateToTransferAccount={onNavigateToTransferAccount}
-            onNavigateToSchedule={onNavigateToSchedule}
-          />
-          <UnexposedCellContent {...props} />
-        </>
-      )}
+      unexposedContent={props => {
+        const payeeInfo = (
+          <div style={{ display: 'flex', alignItems: 'center', width: '100%' }}>
+            <PayeeIcons
+              transaction={transaction}
+              transferAccount={transferAcct}
+              onNavigateToTransferAccount={onNavigateToTransferAccount}
+              onNavigateToSchedule={onNavigateToSchedule}
+            />
+            <UnexposedCellContent
+              {...props}
+              style={{ borderBottom: `1px dashed ${theme.pageTextSubdued}` }}
+            />
+          </div>
+        );
+
+        return importedPayee ? (
+          <Tooltip
+            content={
+              <View style={{ padding: 10 }}>
+                <Text style={{ fontWeight: 'bold' }}>Imported Payee</Text>
+                <Text style={{ fontWeight: 'normal' }}>{importedPayee}</Text>
```

Comment:
Is the normal needed here, shouldn't that already be the case?
```suggestion
                <Text>{importedPayee}</Text>
```

Reply:
Yes, think this left over from a refactor where I had them all in the same line and needed to reset.